### PR TITLE
Added "Copy button" for code example (addon-info)

### PR DESCRIPTION
**Pull request by SuperOl3g**
Issue:
nope

## What I did
Added "Copy button" for code example and add some customization options

### CODE REVIEW

File: addons/info/src/components/markdown/pre/copyButton.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous, { withTheme } from 'glamorous';
+
+const Button = glamorous.button(
+  {
+    outline: 'none',
```
Comment (Hypnosphi) [inline addons/info/src/components/markdown/pre/copyButton.js]: Please either keep the default outline, or provide some replacement for it (i. e. some styles for `:focus` state)

File: addons/info/src/components/markdown/pre/copyButton.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    ...props.styles,
+    transform: props.toggled ? 'translateY(0px)' : 'translateY(-100%) translateY(-6px)',
+  })
+);
+
+function CopyButton(props) {
+  const {
+    copyButton: { toggleText = 'Copied!', text = 'Copy', ...copyButtonStyles } = {},
+    copyButtonContent,
+  } = props.theme;
```
Comment (Hypnosphi) [inline addons/info/src/components/markdown/pre/copyButton.js]: It's quite hard for me to wrap the head around nested destructuring assignments. Can you please make them no more than 1 level deep?

**Comment by Hypnosphi** [comment]
[Live example](https://deploy-preview-2713--storybooks-official.netlify.com/?selectedKind=Addons%7CInfo.React%20Docgen&selectedStory=Comments%20from%20PropType%20declarations&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

**Review by Hypnosphi — CHANGES_REQUESTED** [review]
See comments

**Review by Hypnosphi — APPROVED** [review]

**Comment by Hypnosphi** [comment]
Looks like you need to add `glamorous` to `addons/info/package.json`'s `dependencies`

**Comment by ndelangen** [comment]
🎉

**Comment by SuperOl3g** [comment]
🎊🎊🎊
Can somebody tell when it will be published?)

**Comment by Hypnosphi** [comment]
Probably in a week, @shilman does releases every weekend

**Pull request merged**